tests/python/pants_test/build_graph/test_source_mapper.py

```diff
@@ -38,7 +38,7 @@ def set_mapper(self, fast=False):
     raise NotImplementedError
 
   def owner(self, owner, f):
-    self.assertEqual(set(owner), set(i.spec for i in self.get_mapper().target_addresses_for_source(f)))
+    self.assertEqual({owner}, {i.spec for i in self.get_mapper().target_addresses_for_source(f)})
```

Comment:
`set(owner)` -> `{owner}` might not be right. I think it'll mean that owner will be an element of the set, rather than copied into one.